### PR TITLE
added entry color changes

### PR DESCRIPTION
**Pull request by sanatankc**
Pull request is not complete at the moment. Because I am not able to test them locally.

TODO
- [ ] to test that these changes are actually working and entry is black instead of white.
- [x] Fix white border bug.

### CODE REVIEW

File: gtk3/theme/gtk-widgets.css.em [hunk 1]
```diff
@@ -196,6 +196,10 @@ spinbutton button:last-child,
     border-color: @selection_grey;
 }
 
+spinbutton entry,
+.spinutton.entry {
```
Comment (quozl) [inline gtk3/theme/gtk-widgets.css.em]: You wrote here `spinutton`, presumably a class of elderly sheep that has a habit of hiding in spinifex. :grin:

Ba-aa!

Comment (sanatankc) [reply gtk3/theme/gtk-widgets.css.em]: Oh sorry 😄

**Comment by sanatankc** [comment]
Thanks to @quozl @i5o @walterbender, I finally figured out that changes are actually need to be done in `3.20`. I have removed white border and additionally also fixed buttons height and margin.

screen shot 2017-12-20 at 5 39 39 pm

Weirdly, entry's color is not fixing. I tried
```
spinbutton entry {
    color: #000000;
}
```
Though, setting `background-color`, works.
```
spinbutton entry {
    background-color: #000000;
}
```

**Comment by sanatankc** [comment]
On `entry` no css related to Text is working. I thinks is in different node inside entry node. But can't say what. 

### Alternative
It is bit radical. But if we couldn't able to change text color, why don't we change background of entry.
Like this:
screen shot 2017-12-20 at 9 41 19 pm
Let me know what you guys think about this, @quozl @walterbender @i5o

**Comment by quozl** [comment]
No objection.  Looks cleaner to me.  You might also try changing the - and + foreground colour to differentiate slightly from text foreground colour.

Otherwise, consult with the GTK+ project to figure out how to set the text foreground colour.

**Comment by sanatankc** [comment]
Thanks @quozl, I have updated spin button to differentiate between entry color and buttons.

screen shot 2017-12-21 at 6 48 12 am

**Pull request merged**